Source/Swig/typemap.c

```diff
@@ -1967,8 +1967,9 @@ static void replace_embedded_typemap(String *s, ParmList *parm_sublist, Wrapper
 	    String *parm = Getitem(l, i);
 	    char *eq = strchr(Char(parm), '=');
 	    char *c = Char(parm);
-	    if (eq && (eq - c > 0)) {
-	      String *name = NewStringWithSize(c, (int)(eq - c));
+	    int size = (int)(eq - c);
+	    if (eq && size > 0) {
+	      String *name = NewStringWithSize(c, size);
```

Comment:
You can't use null pointer in pointer arithmetics, this is undefined behaviour whatever you do or don't do with the result later. Pointer arithmetics is only defined for pointers inside the same array.